pkg/metrics/metrics.go

```diff
@@ -129,15 +129,16 @@ var (
 	CSVUpgradeCount = prometheus.NewCounter(
 		prometheus.CounterOpts{
 			Name: "csv_upgrade_count",
-			Help: "Monotonic count of catalog sources",
+			Help: "Monotonic count of CSV upgrades",
 		},
 	)
 )
 
 func Register() {
 	prometheus.MustRegister(csvCount)
-	prometheus.MustRegister(installPlanCount)
-	prometheus.MustRegister(subscriptionCount)
-	prometheus.MustRegister(catalogSourceCount)
+	//TODO: these metrics are actually handled in the catalog operator
```

Comment:
Can we delete these lines instead of commenting them out?

Reply:
You're right, I can do better.